src/panels/config/devices/ha-config-device-page.ts

```diff
@@ -375,6 +432,9 @@ export class HaConfigDevicePage extends LitElement {
     if (Array.isArray(this._diagnosticDownloadLinks)) {
       deviceActions.push(...this._diagnosticDownloadLinks);
     }
+    if (Array.isArray(this._deleteButtons)) {
```

Comment:
It will always be an array, or undefined, so this check can be simpler

Reply:
Makes sense, replaced with:
```diff
-    if (Array.isArray(this._deleteButtons)) {
+    if (this._deleteButtons) {
```